framework/src/org/checkerframework/framework/flow/CFAbstractAnalysis.java

```diff
@@ -115,6 +115,13 @@ public T createTransferFunction() {
     public abstract /*@Nullable*/ V createAbstractValue(AnnotatedTypeMirror type);
```

Comment:
I don't like this design. We now have two abstract methods that each analysis class needs to implement.
I don't see any documentation in this class that makes clear how the two are different and when to use which.
See e.g. NullnessValue and NullnessAnalysis for these new methods that are suddenly necessary.